manifests/tasks/kserve-download-model/README.md

```diff
@@ -0,0 +1,18 @@
+# `kserve-download-model`
+
+This task is used to download a model folder from an S3 bucket. Credentials to allow the download of the model are stored in the `s3-secret` workspace.
+
+## Parameters
+* **model-name**: The name of the folder that contains the model files
+* **s3-bucket-name**: The name of the S3 bucket to be downloaded from
+* **model-relative-path**: The path from the root of the S3 bucket to the folder in which the model folder is located. Passing in an empty value means the model is stored at the root of the bucket
+
+## Workspaces
+* **workspace**: The workspace for the downloaded model
+* **s3-secret**: The workspace containing the S3 credentials needed to download the model
```

Comment:
I think here we could add the format of the s3 secret. It has to be a workspace with a file called `s3-storage-config` (or a ConfigMap with such a key which effectively resolves to such a file) and content like here:
https://github.com/opendatahub-io/ai-edge/blob/94729e1de8d61af47fa823250e16bb9f442298a4/examples/tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template#L15

In this case I would just add it manually to this markdown and not to the yaml definition.